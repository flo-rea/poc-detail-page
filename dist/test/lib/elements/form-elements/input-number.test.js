"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const input_number_1 = require('../../../../src/lib/view/elements/form-elements/input-number');
jest.mock('react-dom');
describe('InputNumber', () => {
    it('has correct default options', () => {
        const component = renderer.create(React.createElement(input_number_1.default, {id: "item2", name: "item2", value: "10", disabled: true}));
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=input-number.test.js.map