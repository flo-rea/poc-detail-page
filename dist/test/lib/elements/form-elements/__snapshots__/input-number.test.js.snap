exports[`InputNumber has correct default options 1`] = `
<input
  disabled={true}
  id="item2"
  name="item2"
  type="number"
  value="10" />
`;
